BonusCalcApi/V1/Infrastructure/BonusPeriod.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace BonusCalcApi.V1.Infrastructure
+{
+    public class BonusPeriod
+    {
+        [Key]
+        public int Id { get; set; }
```

Comment:
These are currently sequence ids but on reflection I wonder whether an ISO 3601 date would be a better id, e.g. `2021-11-01` would be the id for the next bonus period. Similarly, weeks could be identified by the date of the Monday, e.g. `2021-11-01` for week 1, `2021-11-08` for week 2.

@josh-dadswell-hackney @Asura5 wdyt?